packages/apps/docs/src/components/Markdown/index.ts

```diff
@@ -60,4 +61,5 @@ export const markDownComponents: MDXComponents = {
   'kda-youtube': Youtube as FunctionComponent<
     ExtendedIntrinsicElements['kda-youtube']
   >,
+  a: Link as FunctionComponent<ExtendedIntrinsicElements['a']>,
```

Comment:
what would be useful to make a seperate markdown link component.
Check if it is an internal link or external link.

If it is an internal link -> use next/Link so that we stay in the SPA. and not have a hard refresh.